litmus/director/openebs-install/run_litmus_test.yml

```diff
@@ -0,0 +1,93 @@
+---
```

Comment:
i was not able to understand the PR message. What is a connected cluster?

Reply:
Changed PR message to : `Automation of OpenEBS Installation on a K8S Cluster connected to Director Onprem Cluster`